app/models/company.rb

```diff
@@ -127,7 +127,7 @@ def self.address_visible
 
   # TODO: yuck
   def self.with_members
-    where(id: CompanyApplication.where(shf_application: [ShfApplication.where(user: User.current_member)]).pluck(:company_id))
+    where(id: CompanyApplication.where(shf_application: [ShfApplication.where(user: User.members)]).pluck(:company_id))
```

Comment:
@weedySeaDragon I changed this `User` scope because I had failing tests (`problem-scenarios-pagination-view_all_companies`) that were failing because the users created for the test did not actually have membership status (when they should have).  We'll have to adjust certain test steps.

Reply:
I'm confused by your statement:
"the users created for the test did not actually have membership status (when they should have)"

Does that mean that:
1.  you did not create current members?
or 
2.  you _did_ create current members but the system did not recognized them as _current members_ ?

If #1 then the solution is not to revert code so that it is incorrect (the query you changed will now no longer be correct in terms of either business rules or implementation), but to fix your tests.

If #2 then it's a bigger problem and you should create tests (feature or RSpec) that show the problem so that we're all clear on it and can fix it.